web/packages/build/package.json

```diff
@@ -14,25 +14,27 @@
     "@babel/preset-env": "^7.26.0",
     "@babel/preset-react": "^7.25.9",
     "@babel/preset-typescript": "^7.26.0",
+    "@eslint/js": "^9.16.0",
     "@swc/core": "^1.9.3",
     "@swc/plugin-styled-components": "^5.0.0",
     "@types/jsdom": "^21.1.7",
     "@vitejs/plugin-react-swc": "^3.7.2",
     "babel-plugin-styled-components": "^2.1.4",
-    "eslint": "^8.56.0",
+    "eslint": "^9.16.0",
     "eslint-import-resolver-typescript": "^3.6.3",
     "eslint-plugin-babel": "^5.3.1",
     "eslint-plugin-import": "2.31.0",
     "eslint-plugin-jest": "^28.9.0",
     "eslint-plugin-jest-dom": "^5.5.0",
     "eslint-plugin-react": "^7.37.2",
     "eslint-plugin-react-hooks": "^5.0.0",
-    "eslint-plugin-testing-library": "^6.3.0",
+    "eslint-plugin-testing-library": "^7.1.1",
```

Comment:
`pnpm i --resolution-only` was returning warnings about unmet peer deps, so I updated typescript-eslint and eslint-plugin-testing-library.

There's no breaking changes in eslint-plugin-testing-library that would affect us. https://github.com/testing-library/eslint-plugin-testing-library/releases/tag/v7.0.0

<details>
<summary>Output of pnpm i --resolution-only</summary>

```
$ pnpm i --resolution-only
Scope: all 7 workspace projects
 WARN  `node_modules` is present. Lockfile only installation will make it out-of-date
 WARN  12 deprecated subdependencies found: @npmcli/move-file@2.0.1, @types/keyv@4.2.0, abab@2.0.6, are-we-there-yet@3.0.1, boolean@3.1.4, domexception@4.0.0, gauge@4.0.4, glob@7.2.3, glob@8.1.0, inflight@1.0.6, npmlog@6.0.2, rimraf@3.0.2
Progress: resolved 1459, reused 0, downloaded 0, added 0, done
 WARN  Issues with peer dependencies found
web/packages/build
└─┬ eslint-plugin-testing-library 6.3.0
  ├── ✕ unmet peer eslint@"^7.5.0 || ^8.0.0": found 9.16.0
  └─┬ @typescript-eslint/utils 5.62.0
    └── ✕ unmet peer eslint@"^6.0.0 || ^7.0.0 || ^8.0.0": found 9.16.0
```
</details>